quarkus-extension/engine/deployment/src/test/java/org/camunda/bpm/quarkus/engine/test/ManagedJobExecutorTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.camunda.bpm.quarkus.engine.test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.enterprise.inject.Produces;
+import javax.inject.Inject;
+
+import io.quarkus.test.QuarkusUnitTest;
+import org.camunda.bpm.engine.ManagementService;
+import org.camunda.bpm.engine.ProcessEngine;
+import org.camunda.bpm.engine.RepositoryService;
+import org.camunda.bpm.engine.RuntimeService;
+import org.camunda.bpm.engine.cdi.CdiStandaloneProcessEngineConfiguration;
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl;
+import org.camunda.bpm.engine.impl.jobexecutor.JobExecutor;
+import org.camunda.bpm.engine.impl.test.TestHelper;
+import org.camunda.bpm.engine.runtime.ProcessInstance;
+import org.camunda.bpm.quarkus.engine.extension.impl.ManagedJobExecutor;
+import org.eclipse.microprofile.context.ManagedExecutor;
+import org.jboss.shrinkwrap.api.ShrinkWrap;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+public class ManagedJobExecutorTest {
+
+  @RegisterExtension
+  static final QuarkusUnitTest unitTest = new QuarkusUnitTest()
+      .setArchiveProducer(() -> ShrinkWrap.create(JavaArchive.class)
+          .addClass(ExecutorConfig.class)
+          .addAsResource("org/camunda/bpm/quarkus/engine/test/asyncServiceTask.bpmn20.xml"));
+
+  @ApplicationScoped
+  static class ExecutorConfig {
+
+    @Produces
+    public CdiStandaloneProcessEngineConfiguration getCustomProcessEngineConfig() {
+
+      CdiStandaloneProcessEngineConfiguration executorConfiguration = new CdiStandaloneProcessEngineConfiguration();
+      executorConfiguration.setJobExecutorActivate(true);
```

Comment:
Not necessary since `waitForJobExecutorToProcessAllJobs` activates the job executor anyways.